__tests__/app.test.js

```diff
@@ -83,6 +83,43 @@ describe('Reviews  get request test block', () => {
     })
 
 
+
+    test("/api/reviews/3/comments return a list of comments with passed in review_id", () => {
+        return request(app).get("/api/reviews/2/comments").expect(200).then((res) => {
+            const {comments} = res.body;
```

Comment:
When testing each item in an array with a forEach, you need to test the length of the array first. Otherwise. if the array is somehow empty, the forEach will still do it's job and check every item in the array, but because there are no items in the array, it will never return false. This will give a false positive.